packages/cli/package.json

```diff
@@ -6,6 +6,7 @@
     "node": ">=14.15.1"
   },
   "dependencies": {
+    "@coveo/bueno": "^0.1.0",
```

Comment:
That's a generic "parameter validation" library we've built for headless.

I'm using it to validate documents properties in`requiredKeyValidator.ts`

Reply:
Nice reusing!

The more I see it, the more I think we should extract it and put it in its own repo to promote its reusability in @coveo at least

Reply:
Why do you think putting it in it's own repo would make this more re-usable ? It is published as a standalone library on npm, like any other.. 🤔